src/domain/journey/subspace/subspaceHome/SubspaceHomePage.tsx

```diff
@@ -158,7 +157,6 @@ const SubspaceHomePage = ({ dialog }: SubspaceHomePageProps) => {
                     dialogType={SubspaceDialog.Settings}
                     label={t(`spaceDialog.${SubspaceDialog.Settings}` as const)}
```

Comment:
It's not directly related to this PR but using the enum values from SubspaceDialog for translations is not ideal. I think that using the string as 'spaceDialog.settings' is better (for all translations). If we were using the  SubspaceDialog enum to build the translation strings, then it would make sense.